sdk/template-LLC/Azure.Template.LLC/src/autorest.md

```diff
@@ -13,4 +13,8 @@ public-clients: true
 low-level-client: true
 security: AADToken
 security-scopes: https://dev.LLCtemplate.net/.default
+directive:
```

Comment:
Rather than doing this as a transform in the autorest.md file, would you be willing to update the swagger instead?

Reply:
The concern is that other SDKs may have different naming conventions from .NET (For instance, assuming some language would just use `get` and `list` for method names). Changing it in swagger may cause unwanted breaking changes for them?

Reply:
This is a template file used to illustrate to service teams how to use the .NET LLC code generator.  No other languages have dependencies on it, and I think we should use it to show the preferred naming convention for data plane operations.